packages/thirdweb/src/wallets/in-app/web/lib/in-app-account.ts

```diff
@@ -216,6 +231,7 @@ export class IFrameWallet {
             partnerId,
             rpcEndpoint: `https://${tx.chainId}.rpc.thirdweb.com`, // TODO (ew) shouldnt be needed
           },
+          showIframe: isEcosystemWallet(walletId),
```

Comment:
whats the behavior if i set this to true?

Reply:
When it's true it lets the iframe appear to prompt the user's approval for a transaction or signature